### PR TITLE
docs(901-process.md): add Git versioning section

### PR DESCRIPTION
**Pull request by vdice**
* Adds documentation as developed from the proposal in https://github.com/cnabio/cnab-spec/issues/351

Closes https://github.com/cnabio/cnab-spec/issues/351

### CODE REVIEW

**Comment by loot-king** [comment]
Creating a Repository file for my first CLI learning docker

**Comment by vdice** [comment]
@technosophos PTAL when convenient.  Thank you!

**Comment by technosophos** [comment]
Perhaps @silvin-lubecki or @simonferquel could make an "official" LGTM for Chris, whom I suppose we really ought to give LGTM perms to, since he's the director and all. 😁

File: 901-process.md [hunk 1]
```diff
@@ -50,6 +50,34 @@ Finally, certain small errata may be fixed on an existing release without increm
...
+
+### Minor releases
+
+When a minor release of a specification is required, the working group must approve the scope of commits proposed for inclusion. Likely this will be the `master` branch once the approved commit(s) are merged. Next, a new branch should be created from `master` and named `cnab-core-1.1.0-ga` (here assuming that the version immediately prior was `cnab-core-1.0.0`).
+
+When the final release is approved, a Git tag should also be pushed, which triggers schema artifact publishing. Extending the example above, a `cnab-core-1.1.0` tag should be created from the `cnab-core-1.1.0-ga` branch and pushed to origin. We drop the `-ga` suffix as branches and tags may not have the same name in Git.
+
+### Major releases
+
+When a minor release of a specification is required, the working group must approve the scope of commits proposed for inclusion. Likely this will be the `master` branch once the approved commit(s) are merged. Next, a new branch should be created from `master` and named `cnab-core-2.0.0-ga` (here assuming that the version immediately prior was `cnab-core-1.0.0`).
```
Comment (silvin-lubecki) [inline 901-process.md:71]: typo:
> When a ~~minor~~ major release

Comment (vdice) [reply 901-process.md]: Great eye!  Thank you; addressed.

**Pull request merged**